recipes/aws-c-common/all/conanfile.py

```diff
@@ -75,14 +81,18 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "aws-c-common"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "aws-c-common")
+        self.cpp_info.set_property("cmake_target_name", "AWS::aws-c-common")
```

Comment:
A question for modifying other recipes.
Do we need to add `set_property` to the component definition?
Example: `self.cpp_info.components["aws-common-lib"]`.

Reply:
Eventually to avoid to create an unwanted target. @czoido what do you think?

Reply:
Yes, I hope that I'm not missing something, but probably it's better to declare the `cmake_target_name` property for the component as well. Doing this the unwanted target is not created, the target that the consumer will get is the one from the component in the end (there's a check that will avoid the target from the root cpp_info to be created). I think this is the less bad option, and enough to maintain the support for  `cmake_find_package_*` for some time. I'll update the PR with the migration guide and recommend explicitly to set the property on the component as well. cc/ @SpaceIm @toge

Reply:
Thank you for your information!